apps/org/views.py

```diff
@@ -653,3 +655,34 @@ def get_organization(self, request, org_slug):
     def post(self, request, *args, **kwargs):
         from django.http import HttpResponseNotAllowed
         return HttpResponseNotAllowed('GET')
+
+
+class LocalUserAPI(LoginRequiredMixin, View):
+    ''' Setting up a local endpoint for users here. '''
+    def get(self, request, *args, **kwargs):
+        # picking the first organization here
+        org_id = request.user.organization_set.all().first().id
+
+        members = Member.objects.filter(user__organization=org_id)
```

Comment:
Shouldn't have this limitation, as an org may want to request access to a member's data that they haven't worked with until now.